src/instructlab/model/evaluate.py

```diff
@@ -309,6 +314,11 @@ def launch_server(
     type=click.Path(),
     help="Path where all the MMLU Branch tasks are stored. Needed for running mmlu_branch.",
 )
+@click.option(
+    "--gpus",
+    type=click.INT,
+    help="Number of GPUs to utilize for evaluation (not applicable to llama-cpp)",
+)
```

Comment:
My personal preference would be to use `--backend-args` instead of introducing multiple flags. This approach helps us avoid cluttering the CLI with numerous additional flags. Also given that this is a backend-specific flag I don't think it should be exposed directly. `--backend-args` is not present yet (I'm working on changing that). If this gets merged as is, it might eventually get replaced. Just an FYI :)

Reply:
It's probably too late, but did you consider to use subcommands like `ilab model serve vllm` and `ilab model serve llama-cpp`? Then you can have different arguments for each backend.

You could even use `@click.command(context_settings=dict(allow_extra_args=True))` and `ctx.args` to pass additional arguments to vLLM like this:

```console
ilab model serve vllm -- --dtype auto --enable-lora
```

Reply:
It was considered as part of https://github.com/instructlab/dev-docs/pull/81. See this [comment](https://github.com/instructlab/dev-docs/pull/81#issuecomment-2182618445) on why it was discarded.

Reply:
> It's probably too late, but did you consider to use subcommands like `ilab model serve vllm` and `ilab model serve llama-cpp`? Then you can have different arguments for each backend.
> 
> You could even use `@click.command(context_settings=dict(allow_extra_args=True))` and `ctx.args` to pass additional arguments to vLLM like this:
> 
> ```
> ilab model serve vllm -- --dtype auto --enable-lora
> ```

What prevents us from doing this: `ilab model serve --backend vllm -- --dtype auto --enable-lora`?

Reply:
Nothing? :)

Reply:
> My personal preference would be to use `--backend-args` instead of introducing multiple flags. This approach helps us avoid cluttering the CLI with numerous additional flags. Also given that this is a backend-specific flag I don't think it should be exposed directly. `--backend-args` is not present yet (I'm working on changing that). If this gets merged as is, it might eventually get replaced. Just an FYI :)

An argument against folding everything into a --backend-args argument is discoverability.  Since specifying --tensor-parallel-size is necessary to utilize the gpus, it will make discovering the correct way to run with the setting quite difficult without following a document.  And it seems like most users of evaluation will want to use multiple gpus.

Reply:
You could include examples in the doc string of the `serve` function. Click will render the entire doc string in `ilab model serve --help`.